morevna-ep4.msrt

```diff
@@ -296,10 +296,9 @@
 60
 00:09:42,100 --> 00:09:55,000
 [rus] Создание этого эпизода стало возможным благодаря пожертвованиям многих людей со всего мира
-[eng] This episode is created thanks to donations of many people from all around the world
+[eng] This episode was created thanks to the donations of many people from all around the world
 
 61
 00:09:55,240 --> 00:10:11,720
 [rus] Поддержать производство новых эпизодов: https://morevnaproject.org/donate/
-[eng] You can support production of new episodes here: https://morevnaproject.org/donate/
-
+[eng] You can help support production of future episodes here: https://morevnaproject.org/donate/
```

Comment:
"can help support production" - I have feeling that "support" word should be removed, is this correct?

Reply:
No, that was intentional. It does seem rather strange when you look at it literally as support and help can be synonyms, but this is a fairly common phrasing and seems natural to me as a native English speaker. See [this question](https://english.stackexchange.com/questions/225059/to-help-aid-is-it-correct) for a relevant discussion, but basically how I would break it down is that "support" is there to signify that they aren't actually expected to work on the production of the episode, and "help" is there to suggest that a single donation on it's own isn't sufficient to completely support the production of a new episode, it only contributes partially towards that goal.

"help to support" is equally valid as "help support"; there is no difference. The reason for this is unique to help and seems rather complicated, [see here](https://english.stackexchange.com/questions/3578/what-is-the-correct-way-to-use-infinitive-after-the-verb-help-with-or-without) if you are interested. There are also some other good answers in some of the linked questions too.

I did not give any special thought to this line when I wrote it, so thanks for making me think about this more and forcing me to learn about some of these finer details of the English language 🙂

Reply:
Oh, I got that! Thank you for clarification, this makes a lot of sense to me now. ^__^